services/ProductRegistrationService.php

```diff
@@ -88,6 +88,8 @@ private function optInStrategy($email) {
         curl_setopt($curl, CURLOPT_POST, true);
         curl_setopt($curl, CURLOPT_POSTFIELDS, http_build_query(array('email' => $email)));
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
This is a curl module default of 1/true. I believe it has been an issue all along but folks just opted out of registering and was not reported by Ubuntu users.